tests/conftest.py

```diff
@@ -589,6 +589,17 @@ def pytest_addoption(parser) -> None:
         default=False,
         help="check non closed threads i started during test",
     )
+    parser.addoption(
+        "--repeat", action="store", help="Number of times to repeat each test"
+    )
+
+
+def pytest_generate_tests(metafunc):
```

Comment:
I think we also have a library for this here already. Check out the decorator: `@pytest.mark.flaky(reruns=MAX_FLAKY_RERUNS)`

Reply:
alright I see, didn't realize that does the exact same, will remove this then